### PR TITLE
Congratulate Kubernetes on its 10th birthday

### PR DESCRIPTION
**Pull request by sftim**
~I'll update this PR once the CNCF have officially announced the logo for Kubernetes' 10th birthday.~

This PR changes the front page to show a banner about Kubernetes 10th birthday. [Preview](https://deploy-preview-46684--kubernetes-io-main-staging.netlify.app/)

@kubernetes/sig-docs-en-owners @kubernetes/sig-docs-pr-reviews please take a look.

~I'll unhold this once it's good to go.~

/committee steering

### CODE REVIEW

File: content/en/_index.html [hunk 1]
```diff
@@ -8,9 +8,9 @@
 
 {{< site-searchbar >}}
 
-{{< blocks/section id="oceanNodes" >}}
-{{% blocks/feature image="flower" %}}
-[Kubernetes]({{< relref "/docs/concepts/overview/" >}}), also known as K8s, is an open source system for automating deployment, scaling, and management of containerized applications.
+{{< blocks/section class="k8s-overview" >}}
+{{% blocks/feature image="flower" id="feature-primary" %}}
+[Kubernetes]({{< relref "/docs/concepts/overview/" >}}), also known as K8s, is an open-source system for automating deployment, scaling, and management of containerized applications.
```
Comment (justaugustus) [inline content/en/_index.html:13]:
```suggestion
[Kubernetes]({{< relref "/docs/concepts/overview/" >}}), also known as K8s, is an open source system for automating deployment, scaling, and management of containerized applications.
```

ref: https://github.com/cncf/foundation/blob/main/style-guide.md#1-cloud-native-and-open-source

**Review by justaugustus — CHANGES_REQUESTED** [review]

File: content/en/_index.html (hunk 1 above)
Comment (natalisucks) [reply content/en/_index.html:13]: We also added this to our own word list a little while back! https://kubernetes.io/docs/contribute/style/style-guide/#kubernetes-io-word-list

Comment (sftim) [reply content/en/_index.html]: I'll omit the whitespace change here.

File: content/en/community/special/kubernetes-10th-birthday.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Kubernetes' 10th birthday
+content_type: special
+
+display_date_start: 2024-06-06
+display_date_end: 2024-06-15
```
Comment (sftim) [inline content/en/community/special/kubernetes-10th-birthday.md:6]: Is this the end date we'd like?

File: layouts/partials/kubernetes-overview.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{{- $shouldShowBirthdayBanner := false -}}
+{{- $now := time.Now -}}
+{{- with site.GetPage "page" "community/special/kubernetes-10th-birthday" -}}
+  {{- if and (lt ( (.Param "display_date_start") | time.AsTime ) $now ) (gt ( (.Param "display_date_end") | time.AsTime ) ( $now.AddDate 0 0 1 ) ) -}}
+    {{- $shouldShowBirthdayBanner = true -}}
+    {{- warnf "%s" "It's Kubernetes' birthday, fingers crossed this renders OK" -}}
```
Comment (sftim) [inline layouts/partials/kubernetes-overview.html:6]: I don't mind having this warning render for the next 10 days

**Comment by sftim** [comment]
Ready for review! PTAL folks.

File: layouts/partials/kubernetes-overview.html (hunk 3 above)
Comment (katcosgrove) [reply layouts/partials/kubernetes-overview.html:6]: This is cute

**Comment by sftim** [comment]
Unhold this at will, steering.
Unhold this at will, website approvers.

**Review by soltysh — APPROVED** [review]
lgtm with my steering :partying_face: on

**Comment by natalisucks** [comment]
/lgtm
/hold cancel

Thanks for this work, Tim! 🥳 🎈 🎉 

/honk

**Comment by pohly** [comment]
LGTM (steering)

**Pull request merged**